news/_posts/2020-04-27-Polymake.jl-0.4-has-been-tagged.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Polymake.jl 0.4 has been released
+author: Marek Kaluba
+---
+
+`Polymake.jl` 0.4 has been
+[released](https://github.com/oscar-system/Polymake.jl/releases/tag/v0.4.0)!
+
+Since we're upgrading to support `CxxWrap.jl` 0.10.1, we decided to bump the major 
+version to force compilation of libpolymake against libcxxwrap-julia. 
+The release has been somewhat hastened by the need for
+[`Oscar.jl`](https://github.com/oscar-system/Oscar.jl) jointly working with 
+the newest `CxxWrap.jl`, so expect 0.4.1 and later minor versions rather heavy
+in added features :).
+
+<br/>
+_Note_:
+We're tracking the version of libcxxwrap-julia used, so that we can warn you
+whenever executing `import Pkg; Pkg.build("Polymake")` is necessary, but some of the cases
+might have slipped us. If you experience crashes while `using Polymake` try:
+ * `import Pkg; Pkg.free("CxxWrap")`,
+ * `import Pkg; Pkg.build("Polymake")`,
+ * [filing a bug report!](https://github.com/oscar-system/Polymake.jl/issues)
+
+Below you can find a summary of the major changes and design decision we have 
+taken in the development of `Polymake.jl`.
+
+<br/>
+Happy coding,<br/>
+on behalf of polymake Team,<br/>
+Marek
+
+# Changes between Polymake.jl 0.3 and 0.4
+
+## Library updates
+
+* `Polymake.jl` can now be used with `CxxWrap` 0.10.1 or later
+* `Polymake.jl` uses `LoadFlint.jl` and latest `SingularBuilder`
+
+## New features
+
+* documentation strings of all `polymake` functions and methods was made 
+available in julia 
+* handling of polynomials should be much smoother
+* polymake banner is suppressed when imported in `Oscar.jl`
+
+## Fixed notable bugs
+
+* `libpolymake` no longer requires GLIBCXX_3.4.26, i.e. `Polymake.jl` is 
+compatible with libstdc++ shipped by julia, even when using very modern Linux 
+systems (big thanks to Max Horn for investigating!)
+* polymake doesn't grab all interrupts, so after `using Polymake` one can 
```

Comment:
Not sure if it is intentionally that sometimes `polymake` is in a code block and sometimes not (there are valid reasons to do that, of course; e.g. for GAP, I'll write "GAP" for the system, but "`gap`" if I refer to the executable). I don't see a pattern here, but of course that doesn't mean there is one. So please take this suggestion strictly optional (similar for "polymake banner" in line 46).
```suggestion
* `polymake` doesn't grab all interrupts, so after `using Polymake` one can 
```